Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM adzerk/boot-clj:latest as builder
+
+ENV BOOT_VERSION=2.7.2
+ENV BOOT_CLOJURE_VERSION=1.8.0
+
+WORKDIR /usr/local/github-changelog
+
+COPY . .
+RUN cd /usr/local/github-changelog && boot uberjar
+
+RUN /bin/bash -c 'source /usr/local/github-changelog/version.properties && cp /usr/local/github-changelog/target/github-changelog-$VERSION.jar /usr/local/github-changelog/github-changelog.jar'
```

Comment:
This should be a `mv` to make it faster.

Reply:
Fixed